static/sass/_pattern_cve.scss

```
@@ -302,4 +302,20 @@
       width: 100%;
     }
   }
+
+  .cve-filters {
+    .p-side-navigation {
+      #tab1-section {
+        margin-top: 1rem;
+      }
+
+      #affectedPackages {
+        width: auto;
+      }
+    }
```

```suggestion
  .cve-filters {
    .p-side-navigation {
      #tab1-section {
        margin-top: 1rem;
      }

      #affectedPackages {
        width: auto;
      }
    }
```
We should try and only use classes for targeting styles, as the id's could change in the future. The psuedo class `[:first-child](https://developer.mozilla.org/en-US/docs/Web/CSS/:first-child)` might be useful here